samples/arrays/array_bad2.jakt

```diff
@@ -1,5 +1,5 @@
 /// Expect:
-/// - error: "Index is not an integer"
+/// - error: "Index must be an Integer or a Range"
```

Comment:
these shouldn't be capitalised, unless that's the name of the type

Reply:
Sure, have updated this.